src/styles/main.scss

```diff
@@ -1,3 +1,249 @@
-body {
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap);
+
+$main-color:#00acdc;
+$header-height: 60px;
+$white: #fff;
+$title-color: #060b35;
+$text-secondary-color: #616070;
+$card-margin: 16px;
+$logo-size: 40px;
+$star-size: 16px;
```

Comment:
I prefer to use `color` as a prefix (`$color-main`, `$color-title`, etc.). But it's optional.